charts/mirrorbits-parent/tests/custom_values_ingress_test.yaml

```
@@ -3,20 +3,23 @@ set:
   ## Mock subcharts default values
   mirrorbits-lite:
     enabled: true
+    backendServiceNameTpl: '{{ default "RELEASE-NAME-mirrorbits-lite" }}'
     service:
       port: 7777
   httpd:
     enabled: true
+    backendServiceNameTpl: '{{ default "RELEASE-NAME-httpd" }}'
     service:
       port: 8080
   ## End of subchart mocked values
-  ingress:
-    enabled: true
-    hosts:
-      - host: company.org
-        paths:
-          - path: /
-            backendService: mirrorbits
+  global:
+    ingress:
+      enabled: true
+      hosts:
+        - host: company.org
+          paths:
+            - path: /
+              backendService: mirrorbits
```

```suggestion
              backendService: mirrorbits-lite
```
(Depends on the acceptance or not of the first suggestions above)